src/Notifications/NotificationCenterController.cs

```diff
@@ -21,6 +23,8 @@ public class NotificationCenterController
         private static readonly string htmlEmbeddedFile = "Dynamo.Notifications.node_modules._dynamods.notifications_center.build.index.html";
         private static readonly string jsEmbeddedFile = "Dynamo.Notifications.node_modules._dynamods.notifications_center.build.index.bundle.js";
 
+        private event Action<ILogMessage> MessageLogged;
```

Comment:
not sure this will do anything until you 
1. implement ILogSource and
2. I think Dynamo only pays attention to `ILogSource` objects if they are extensions - so you could relay these messages through the notification extension and make it implement `ILogSource` - that should get hooked up automatically since it's a ViewExtension.

Reply:
Ok so implementing `ILogSource` in notification extension and then relaying the object in controller to log messages then?